pkg/build/controller/common/util.go

```diff
@@ -134,3 +140,46 @@ func HasBuildPodNameAnnotation(build *buildapi.Build) bool {
 	_, hasAnnotation := build.Annotations[buildapi.BuildPodNameAnnotation]
 	return hasAnnotation
 }
+
+// ProcessValueFrom supports the k8s pod syntax of allowing "valueFrom" in addition
+// to "value", where "valueFrom" uses the downward api to reference existing env
+// variables, jsonpath references to the build object, secrets, and configmaps.
+func ProcessValueFrom(pod *kapi.Pod, client kclientset.Interface) error {
```

Comment:
Process doesn't feel like the right name for something that's actually mutating the Pod that's passed in.